settings.gradle.kts

```diff
@@ -156,7 +154,7 @@ pluginManagement {
                     useModule("com.android.tools.build:gradle:3.5.3")
 
                 pluginId.startsWith("org.jetbrains.kotlin.") ->
-                    useVersion(kotlinVersion)
+                    useVersion("1.3.70")
```

Comment:
Why do we need that hardcode?

Reply:
Dependencies.kt is not available in settings.gradle.kts